src/app.ts

```diff
@@ -100,6 +100,15 @@ async function app() {
     return;
   }
 
+  const fhirR4Dependency = config.dependencies['hl7.fhir.r4.core'];
+  if (!(fhirR4Dependency && fhirR4Dependency === '4.0.1')) {
```

Comment:
Does this need the check that `fhirR4Depency` exists? Doesn't the equality check that for us? Not a huge deal either way.

Also do we necessarily not allow using `4.0.0`?

Reply:
I take back the second question I think. From what I can tell, you can't even get `4.0.0` off of the fhir package server. If you go to http://packages.fhir.org/hl7.fhir.r4.core only the `4.0.1` version is listed. But if you go to http://packages.fhir.org/hl7.fhir.us.core, you can see a bunch of different versions. So if they aren't posting `4.0.0` as a package, then people probably shouldn't be using it I'm thinking?

Reply:
Exactly. At first I was checking that they specified a `4.0.x` version, but then noticed the same thing you noticed on packages.fhir.org. So I mentioned it to Chris at my 1 on 1 yesterday, and he said we should just require that they use `4.0.1`.